prettier.config.js

```diff
@@ -1,5 +0,0 @@
-module.exports = {
```

Comment:
I don't know if deleting prettier.config.js is a good idea. Because in future changes the code will be indented with the configuration of each code editor?

Reply:
Configuration was moved to `.eslintrc.js` file.
I've got a prettier plugin in my editor and it was recognizing the config correctly. After this change `yarn lint` will let you know if something is wrong even if you don't have a plugin. Also CI (`yarn validate`) will be double-checking that everything is ok.